workspace/utilities/lib/util-oembed-attr.xsl

```diff
@@ -0,0 +1,26 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<xsl:stylesheet version="1.0" xmlns:xsl="http://www.w3.org/1999/XSL/Transform">
+
+<xsl:template name="util-oembed-attr">
+	<xsl:param name="driver" select="driver" />
+	<xsl:param name="url" select="url" />
+	<xsl:param name="id" select="@id" />
+	<xsl:param name="title" select="title" />
+
+	<set>
+		<xsl:attribute name="data-oembed-id">
+			<xsl:choose>
```

Comment:
Move chose outside.. choose between one set or other.. dont use xsl:attribute

Reply:
This was a copy/pasta from old code. Changed it!